### ISSUE TITLE
kube-proxy cleanup command fails

### ISSUE THREAD

**Issue by zerkms** [cloudnativelabs/kube-router#760]
Kubernetes version: 1.15.1

```
# docker run --privileged -v /lib/modules:/lib/modules --net=host k8s.gcr.io/kube-proxy-amd64:v1.10.2 kube-proxy --cleanup
Unable to find image 'k8s.gcr.io/kube-proxy-amd64:v1.10.2' locally
v1.10.2: Pulling from kube-proxy-amd64
d093dc0d702b: Pull complete 
14bfddfd7fdf: Pull complete 
e37e9b622b62: Pull complete 
Digest: sha256:c7036a8796fd20c16cb3b1cef803a8e980598bff499084c29f3c759bdb429cd2
Status: Downloaded newer image for k8s.gcr.io/kube-proxy-amd64:v1.10.2
W0723 01:11:16.944724       1 server.go:195] WARNING: all flags other than --config, --write-config-to, and --cleanup are deprecated. Please begin using a config file ASAP.
I0723 01:11:16.944845       1 feature_gate.go:226] feature gates: &{{} map[]}
I0723 01:11:16.962496       1 server.go:444] Version: v1.10.2
E0723 01:11:17.063216       1 proxier.go:449] Failed to execute iptables-restore for filter: exit status 1 (iptables-restore: line 5 failed
)
error: encountered an error while tearing down rules.
```

**Comment by murali-reddy** [comment]
Not sure why kube-proxy failed to cleanup. May be try the latest image version of cube-proxy.

docker run --privileged -v /lib/modules:/lib/modules --net=host k8s.gcr.io/kube-proxy-amd64:v1.15.1 kube-proxy --cleanup

**Comment by zerkms** [comment]
Right, I should have mentioned that the command was taken from the docs: https://github.com/cloudnativelabs/kube-router/blob/master/docs/kubeadm.md

**Comment by zerkms** [comment]
```
# docker run --privileged -v /lib/modules:/lib/modules --net=host k8s.gcr.io/kube-proxy-amd64:v1.15.1 kube-proxy --cleanup
Unable to find image 'k8s.gcr.io/kube-proxy-amd64:v1.15.1' locally
v1.15.1: Pulling from kube-proxy-amd64
Digest: sha256:5e525d1b5cf33a721768697bde0f20baa3f6967c3a10860617a15ec7cbb586d7
Status: Downloaded newer image for k8s.gcr.io/kube-proxy-amd64:v1.15.1
W0723 22:12:41.842429       1 server.go:216] WARNING: all flags other than --config, --write-config-to, and --cleanup are deprecated. Please begin using a config file ASAP.
F0723 22:12:42.199311       1 server.go:449] <nil>
```

this time it succeeded yes.

I believe the documentation should be updated then

**Comment by murali-reddy** [comment]
fixed in#762

**Comment by marczis** [comment]
I have the same issue with v1.15.1 ... do I need a newer one?
